src/components/BarChart/BarChart.tsx

```diff
@@ -238,23 +241,17 @@ export const BarChart = ({
     [activeBar, loading, rangeMax, rangeMin, value]
   )
 
-  const timeoutRef = useRef<NodeJS.Timeout>()
-
   // disables the behavior of chanhing value while scrolling on top of the input
   const onRangeWheel = useCallback((e) => {
     // Prevent the input value change
     e.target.blur()
     // Prevent the page/container scrolling
     e.stopPropagation()
-    timeoutRef.current = setTimeout(() => {
+    timeout.current = setTimeout(() => {
       e.target.focus()
     }, 0) as unknown as NodeJS.Timeout
   }, [])
 
-  useEffect(() => {
```

Comment:
I already had a `useEffect` to clear another timeout in the un-mount phase 😂 . I'm re-using that ref now. Here: https://github.com/decentraland/ui/pull/294/files#diff-59a21f74a5514ba2dd1fa23bcbaa43e53be9f00352cef5cbaadf5907a6e715d5R68